lua/ulx/modules/sh/cfc_propify.lua

```diff
@@ -0,0 +1,208 @@
+CFCUlxCommands.Propify = CFCUlxCommands.Propify or {}
+local cmd = CFCUlxCommands.Propify
```

Comment:
Same reason as before
```suggestion
CFCUlxCommands.propify = CFCUlxCommands.propify or {}
local cmd = CFCUlxCommands.propify
```